algorithms/48/fib.cpp

```diff
@@ -1,11 +1,35 @@
 #include <iostream>
+#include <bits/stdc++.h>
```

Comment:
Don't include this. It includes unnecessary header files (yes we do use it in competitive programming since it's cumbersome to add header files separately in CP, but it's not a good programming practice)

Reply:
removed unneeded header